test/e2e/flask/user-operations.spec.ts

```diff
@@ -210,7 +210,7 @@ async function withAccountSnap(
       useBundler: true,
       usePaymaster: Boolean(paymaster),
       dapp: true,
-      ganacheOptions: {
+      localNodeOptions: {
```

Comment:
in all the specs where we were using custom ganacheOptions, we now rename the ganacheOptions, for localNodeOptions. This way, we don't care which node we are using  (anvil, ganache, bitocin..), this can be left as generic name and use in whichever node we chose to select